third_party/crashpad/minidump/BUILD.gn

```diff
@@ -93,6 +93,25 @@ static_library("minidump") {
   }
 }
 
+# :format is the only part of minidump that snapshot may depend on.
+static_library("format") {
+  sources = [
+    "minidump_context.h",
+    "minidump_extensions.cc",
+    "minidump_extensions.h",
+  ]
+
+  public_configs = [ "..:crashpad_config" ]
+
+  public_deps = [ "../compat" ]
+
+  deps = [
+    "../snapshot:context",
+    "../third_party/mini_chromium:base",
+    "../util",
+  ]
+}
+
 static_library("test_support") {
   testonly = true
```

Comment:
This `testonly` was a Cobalt customization that was removed in https://lbshell-internal-review.git.corp.google.com/c/cobalt_src/+/264151. But I guess the new version of Chromium had it?

Reply:
You don't have to guess!
https://chromium.googlesource.com/chromium/src/+/84.0.4147.144/third_party/crashpad/crashpad/minidump/BUILD.gn#110